.pre-commit-hooks.yaml

```diff
@@ -115,7 +115,7 @@
   name: Run MegaLinter (skipping linters that run in project mode)
```

Comment:
Commit message (nit): "The previous [...] syntax, but [...] syntax, and [...]"

Reply:
Thank you! I'm realizing perhaps I should also change "name" to "names" both places that appears.